pkg/info/infopb/rpc.proto

```diff
@@ -56,9 +56,13 @@ message StoreInfo {
     int64 min_time = 1;
     int64 max_time = 2;
     bool  supports_sharding = 3;
+    // TODO(fpetkovski): Remove in v1.0
     bool  sends_sorted_series = 4;
+    // TODO(fpetkovski): Remove in v1.0
+    bool  sends_sorted_series_without_labels = 5;
```

Comment:
Adding a new field here because `sends_sorted_series` does not mean the same as sending series sorted without certain labels.